### PR TITLE
Add Katello 4.10

### PR DESCRIPTION
**Pull request by chris1984**
@ekohl how does this look?

### CODE REVIEW

File: releases/katello/4.10/settings [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FULLVERSION='4.10.0'
```
Comment (ekohl) [inline releases/katello/4.10/settings:1]: This implies it's GA, but the full version includes the RC part. See dbc04052116c8e58d1323b9894444a0802f5db3f

Comment (chris1984) [reply releases/katello/4.10/settings]: @ekohl fixed, sorry about that I should have looked at a rc1 commit before just blindly copy/pasta

Comment (ekohl) [reply releases/katello/4.10/settings]: For Foreman's branching procedure we have explicit instructions: https://github.com/theforeman/theforeman-rel-eng/blob/5f6403570c60b7144168439562d9b5a3d722de09/procedures/foreman/branch.md.erb#L62

Katello should have a similar step, which would prevent any future confusion. Could you submit a PR?

**Review by ekohl — APPROVED** [review]

File: releases/katello/4.10/settings (hunk 1 above)
Comment (chris1984) [reply releases/katello/4.10/settings]: Yeah will make a follow up today

**Pull request merged**